app/templates/request-code-sent-by-email.html

```diff
@@ -0,0 +1,58 @@
+{%- extends 'base-' + display_region + '.html' -%}
+
+{%- set start_study_link = url('Start:get', display_region=display_region) -%}
+{%- set request_code_link = url('RequestEnterAddress:get', display_region=display_region, user_journey='request', request_type=request_type) -%}
+
+{%- from 'components/button/_macro.njk' import onsButton -%}
+{%- from "components/panel/_macro.njk" import onsPanel -%}
+
+{%- block main -%}
+
+    {%-
+        call onsPanel({
+            'type': 'success',
+            'id': 'success-id',
+            'iconType': 'check',
+            'iconSize': 'l',
+            'classes': 'ons-u-mt-xl ons-u-mb-m'
+        })
+    -%}
+
+        {%- autoescape false -%}
+            <h1 class="ons-u-mb-xs ons-u-fs-l">{{ _('An email has been sent to %(email)s', email=email)|safe }} </h1>
+        {%- endautoescape -%}
+
+        <p>{{ _('The email message with a new access code should arrive soon for you to start your study') }}</p>
+
+    {%- endcall -%}
+
+    {%-
+        call onsPanel({
+            'type': 'bare',
+            'iconType': 'lock',
+            'classes': 'ons-u-mb-l'
+        })
+    -%}
+
+        <p>{{ _('The email will be sent from <strong>@ons.gov.uk</strong>') }}</p>
+
+    {%- endcall -%}
+
+    <h2 class="ons-u-fs-m ons-u-mb-xs">{{ _("Didn’t receive an email?") }}</h2>
+
+    {%- autoescape false -%}
+        <p class="ons-u-mb-l">{{ _("It can take a few minutes for the email to arrive. If it doesn’t arrive, you can %(open)srequest a new access code%(close)s.", open='<a href="%s">' % request_code_link, close='</a>')|safe }}</p>
```

Comment:
Should there be a space between '%(open)srequest'?

Reply:
No - the open is the start of the link, so doesn't need a additional space

Reply:
Ah got you.